tests/build/net45/NLuaTest.csproj

```diff
@@ -45,6 +45,7 @@
       <HintPath>..\..\..\packages\NUnit.3.12.0\lib\net45\nunit.framework.dll</HintPath>
     </Reference>
     <Reference Include="System" />
```

Comment:
Why do we need the System.Data to test this? Can't we use a POCO object? NLua tests will run on Linux, Mac including iOS and tvOS.

Reply:
Good point.  I will fix this to use one of the test classes.

Reply:
So I changed to use the test class but Mac is still failing.  Do I need ` import ('mscorlib','System')` in DoString? I'm not sure what I am doing wrong.